src/ssz/type/container.zig

```diff
@@ -74,6 +76,13 @@ pub fn FixedContainerType(comptime ST: type) type {
             return true;
         }
 
+        /// Creates a new `FixedContainerType` and clones all underlying fields in the container.
+        ///
+        /// Caller owns the memory.
+        pub fn clone(value: *const Type, out: *Type) !void {
+            inline for (fields) |field| @field(out, field.name) = @field(value, field.name);
```

Comment:
can the implementation here match the one in byte_vector?

Reply:
doable for the fixed types, simplifies implementation quite a bit. Done in [1898a85](https://github.com/ChainSafe/ssz-z/pull/51/commits/1898a85062cc0f3610ca7e3069a92dda160bb6af)